### PR TITLE
feat: Ingest ImageLA content hub

### PR DESCRIPTION
**Pull request by KevinJBoyer**
## Ticket

https://navalabs.atlassian.net/browse/DST-584

## Changes
 - Add offline scraper (requires running a headless browser) for downloading ImagineLA content hub HTML
 - Add ingestion task that parses content hub HTML files locally or in S3

## Context for reviewers
 - Our docker image is slim and so does not contain the dependencies for running a headless browser

## Testing
 - Added test coverage in test_ingest.py

### CODE REVIEW

File: app/Makefile [hunk 1]
```diff
@@ -259,3 +259,6 @@ ingest-bem-pdfs: check-ingest-arguments
 
 ingest-edd-web: check-ingest-arguments
 	$(PY_RUN_CMD) ingest-edd-web $(DATASET_ID) $(BENEFIT_PROGRAM) $(BENEFIT_REGION) $(FILEPATH)
+
+ingest-imagine-la: check-ingest-arguments
+	$(PY_RUN_CMD) ingest-imagine-la $(DATASET_ID) $(BENEFIT_PROGRAM) $(BENEFIT_REGION) $(FILEPATH)
```
Comment (yoomlam) [inline app/Makefile:264]: Merge with `main` and add quotes around args

File: app/pyproject.toml [hunk 2]
```diff
@@ -71,7 +71,7 @@ ingest-guru-cards = "src.ingest_guru_cards:main"
 ingest-policy-pdfs = "src.ingest_policy_pdfs:main"
 ingest-bem-pdfs = "src.ingest_bem_pdfs:main"
 ingest-edd-web = "src.ingest_edd_web:main"
-scrape-edd-web = "src.ingestion.scrape_edd_web:main"
```
Comment (yoomlam) [inline app/pyproject.toml:74]: Did you intend to delete this line?

File: app/src/ingestion/imagine_la/scrape/scrape_content_hub.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# /// script
+# dependencies = [
+#   "install-playwright",
+#   "playwright",
+# ]
+# ///
```
Comment (yoomlam) [inline app/src/ingestion/imagine_la/scrape/scrape_content_hub.py:6]: What are these lines for? Is this for `uv` to use?

File: app/src/ingestion/imagine_la/scrape/scrape_content_hub.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+and saves them to .html files in the `pages` subdirectory.
+
+It is intended to be run locally: the HTML files can later be added
+to S3 to ingest them into a deployed environment's database.
+
+You can either install the above dependences with pip, e.g.,
+`pip install -r requirements.txt` before running with
+`python scrape_content_hub.py`, or run this with
+`uv run --no-project scrape_content_hub.py` to have an environment
+automatically created for you.
```
Comment (yoomlam) [inline app/src/ingestion/imagine_la/scrape/scrape_content_hub.py:19]: Is this a pattern we should follow for password-protected websites?

File: app/pyproject.toml (hunk 2 above)
Comment (KevinJBoyer) [reply app/pyproject.toml:74]: ...No I did not, I thought I was deleting my own line that I added earlier in development 🤦  thanks for catching!

File: app/src/ingestion/imagine_la/scrape/scrape_content_hub.py (hunk 3 above)
Comment (KevinJBoyer) [reply app/src/ingestion/imagine_la/scrape/scrape_content_hub.py:6]: Yes, I will add comments to clarify -- I left in the requirements file so folks could use pip, but `uv` is really nice for these single-file scripts (`uv run scrape_content_hub.py` will very quickly create a virtual env with these packages and execute the script, then clean up once its done)

**Pull request merged**